### PR TITLE
[build-script] Add --time-test to build-script invocation for lldb te…

### PR DESCRIPTION
**Pull request by dcci**
…sts.

<rdar://problem/48986159>

### CODE REVIEW

**Comment by dcci** [comment]
@swift-ci test

File: utils/build-script-impl [hunk 1]
```diff
@@ -3122,7 +3122,7 @@ for host in "${ALL_HOSTS[@]}"; do
                     with_pushd ${results_dir} \
                         call "${llvm_build_dir}/bin/llvm-lit" \
                              "${lldb_build_dir}/lit" \
-                             ${LLVM_LIT_ARGS} \
+                             ${LLVM_LIT_ARGS} -v --time-tests \
```
Comment (adrian-prantl) [inline utils/build-script-impl]: What about amending LLVM_LIT_ARGS only once on line 3116?

**Review by adrian-prantl — CHANGES_REQUESTED** [review]

File: utils/build-script-impl (hunk 1 above)
Comment (adrian-prantl) [reply utils/build-script-impl]: or even earlier, since we probably want this to affect every lit invocation?

**Review by adrian-prantl — APPROVED** [review]

File: utils/build-script-impl (hunk 1 above)
Comment (dcci) [reply utils/build-script-impl]: got it.

**Review by shahmishal — APPROVED** [review]

**Comment by dcci** [comment]
@swift-ci test and merge

**Comment by dcci** [comment]
```
/home/buildnode/jenkins/workspace/swift-PR-Linux/branch-master/swiftpm/Sources/PackageDescription4/Package.swift:12:8: error: missing required module 'SwiftOverlayShims'
14:51:18 import Glibc
14:51:18        ^
```

is this a known issue?

**Comment by shahmishal** [comment]
@dcci Yes. 

/cc @Rostepher

**Pull request merged**